src/api/getWilayaByCode.js

```diff
@@ -0,0 +1,19 @@
+const projectWilaya = require('../utils/projections/wilayaProjection');
+
+const getWilayaByCode = data =>
+/**
+ * Takes a wilaya code (matricule) and returns its respective wilaya
+ *
+ * @example <p> Get Oran wilaya
+ * //returns ["Oran"]
+ * getWilayaByCode(31)
+ *
+ * @param { Number} mattricule The Wilaya's "matricule"
+ * @returns { Number[] | undefined } Returns matching wilaya, or undefined
```

Comment:
This returns either an object or undefined.

```suggestion
 * @returns { Object | undefined } Returns matching wilaya, or undefined
```